src/Symfony/Component/DependencyInjection/Compiler/PassConfig.php

```diff
@@ -57,9 +57,9 @@ public function __construct()
         );
 
         $this->removingPasses = array(
+            new ReplaceAliasByActualDefinitionPass(),
             new RemovePrivateAliasesPass(),
             new RemoveAbstractDefinitionsPass(),
-            new ReplaceAliasByActualDefinitionPass(),
```

Comment:
This looks like a bug fix to me, is it? Aliases should be resolved before removing private and abstract definitions, isn't it?